### DISCUSSION TITLE
tag label syntax?

### DISCUSSION THREAD

**Discussion by pjotrek-b** [minio/minio#19464] [Q&A] [answered]
I want to use a tag called "keywords" and tried entering "one,two,three four" and so on. Didn't let me "save" it.

I couldn't find documentation on which characters are allowed.
Where could I find it, and is it possible to change that? (theoretically allow all unicode characters, arbitrary string-size)

Thank you!

**Comment by harshavardhana** [comment, marked as answer]
You cannot. Its an S3 API limit, ASCII characters only for tags.

**Reply by pjotrek-b** [reply]
Alright.
Thanks for the quick reply!

Being aware of the consequences:
Could I technically (for an in-house prototype) ignore-and-break the S3 API compatibility and change Minio's code (or API libs) to change that?
In what files/libs (approximately) would these limitations be set?


Thank you very much. :nerd_face:

**Reply by harshavardhana** [reply]
you can do what you want, but SDKs may not work properly.

**Reply by pjotrek-b** [reply]
Understood.
Thank you!
